hudi-client/hudi-client-common/src/main/java/org/apache/hudi/io/HoodieCreateHandle.java

```diff
@@ -180,28 +180,34 @@ public IOType getIOType() {
       fileWriter.close();
 
       HoodieWriteStat stat = new HoodieWriteStat();
-      stat.setPartitionPath(writeStatus.getPartitionPath());
-      stat.setNumWrites(recordsWritten);
-      stat.setNumDeletes(recordsDeleted);
-      stat.setNumInserts(insertRecordsWritten);
-      stat.setPrevCommit(HoodieWriteStat.NULL_COMMIT);
-      stat.setFileId(writeStatus.getFileId());
-      stat.setPath(new Path(config.getBasePath()), path);
       long fileSizeInBytes = FSUtils.getFileSize(fs, path);
       stat.setTotalWriteBytes(fileSizeInBytes);
       stat.setFileSizeInBytes(fileSizeInBytes);
-      stat.setTotalWriteErrors(writeStatus.getTotalErrorRecords());
```

Comment:
The only difference between `HoodieCreateHandle` and `FlinkCreateHandle` is how they fetch the fize size, can we extract that part out ? Because usually we override the different part.

Reply:
@danny0405 Thank you for your suggestions. I made some changes, please take a look.